src/app/screens/Dashboard/screens/Home/utils.js

```diff
@@ -0,0 +1,7 @@
+export const getComponentsCode = components =>
+  components.map(comp => ({
+    title: comp.name,
+    html: comp.object.entries[0].object.text,
+    css: comp.object.entries[1].object.text,
+    scss: comp.object.entries[2].object.text
+  }));
```

Comment:
Nice!

Though you could receive an empty array from entries. Also, we could make this a TS file.

```js
export const getComponentsCode = components =>
  !!components?.length ? components.map(comp => ({
    title: comp.name,
    html: comp.object.entries[0].object.text,
    css: comp.object.entries[1].object.text,
    scss: comp.object.entries[2].object.text
  })) : [];
```